cli/cmd/policy_create.go

```diff
@@ -92,6 +135,12 @@ func createPolicy(cmd *cobra.Command, _ []string) error {
 	if cli.JSONOutput() {
 		return cli.OutputJSON(createResponse.Data)
 	}
+	if queryExists {
+		cli.OutputHuman(fmt.Sprintf("The query %s already exists.\n", newPolicy.QueryID))
```

Comment:
Should we call this out? 🤔 I don't think we should (my own opinion). I would make it a log message

Reply:
Well we're communicating that we're creating a query and a policy and if we don't output anything about the query it might be a wtf moment...

Reply:
Going to merge this down, but let me know what you prefer here and we can still make it happen.